modules/src/ics04_channel/channel.rs

```diff
@@ -113,6 +113,7 @@ impl ChannelEnd {
     pub fn set_state(&mut self, s: State) {
         self.state = s;
     }
+
     pub fn set_version(&mut self, v: String) {
         self.version = v;
     }
```

Comment:
could you please add 

```
pub fn set_counterparty_channel_id(&mut self, c: ChannelId) {
        self.remote.channel_id = Some(c);
    }
```


Reading I realized it's required in chan_open_ack

Reply:
Thanks, should be fixed in fcf6c371.